src/LinkMgrdMain.cpp

```diff
@@ -59,6 +59,7 @@ namespace {
 int main(int argc, const char* argv[])
 {
     int retValue = EXIT_SUCCESS;
+    int dummyVal = -1;
```

Comment:
why do we need this?

Reply:
I was trying to trigger the lgtm check with some code change. Will remove it soon.